create/application_test.go

```diff
@@ -16,12 +18,60 @@ import (
 	"github.com/ninech/nctl/api/util"
 	"github.com/ninech/nctl/internal/test"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/utils/pointer"
 )
 
+const (
+	// dummySSHPrivateKey is a dummy private SSH key with some whitespace
+	// around
+	dummySSHPrivateKey = `
```

Comment:
Nit: could have used an `ed25519` key so it would be a bit smaller :)

Or rather, maybe we should test both to make sure the validation works?

Reply:
Good idea...will do so.

Reply:
done